scripts/dictquery.py

```
@@ -0,0 +1,15 @@
+import functools
+
+
+def dictquery(input_dict, listofpath):
+    """Fetches data from nested dictionary
+    Args:
+        input_dict: nested Dictionary
+        listofpath: path of desired value
+    Returns:
+        list of values at path specified by input list
+    """
+    output_list = []
+    for path in listofpath:
+        output_list.append(functools.reduce(dict.get, path.split('/'), input_dict))
```

Nice use of `functools.reduce`

Doesn't this keep making `dict.get` calls if the call fails early on in path? example:
```python
dictquery({}, ['rekt/gg/no/re'])
```
will make some calls, then fail when dict,get reruns `None` and None is attempted to be used again.

---

I didn't test the fail state...  I'll write a test and figure that out.  Still, that's probably what'll happen.  A couple unnecessary dict.get calls.